### ISSUE TITLE
Abandoned project?

### ISSUE THREAD

**Issue by ghost** [maierfelix/POGOserver#319]
What happened?

Is there any other project of this type?

**Comment by ghost** [comment]
Its not abandones its slow as the developers have other stuff going on im sure. Last i heard there was alot to figure out to update to new versions

**Comment by ghost** [comment]
@scorp12 Yes, I saw that.

I think the packets are encrypted, I tried to do a server in the current version more without success.

I think the packets are encrypted.

**Comment by ghost** [comment]
Again not alot try to help out so slows things dowm

**Comment by gabriel4dev** [comment]
I tried to setup this server and make it work without success.

**Comment by ghost** [comment]
Whar os are u using

**Comment by bleonard252** [comment]
Yes this is an abandoned project. It has not been updated since last year.

**Comment by chaze117** [comment]
The main problem is the client-side protection, when you modify your OS on your phone what is required for connect to a custom server PGO will stop working. If I remember well Safetynet protection, I think I can't be removed or bypassed very well. 

And yeah I think this project is abandoned for a long time. It will be great to see a comeback to this project.

**Comment by ghost** [comment]
Yeah. Seems project over write projects and are forgotten

**Comment by chaze117** [comment]
Maybe something for bypassing the safetynet protection? The only thing we need is a working method to connect custom endpoint.
